### PR TITLE
Prepare the 3.1.3 release blogpost

### PR DESCRIPTION
**Pull request by Kordyjan**
While still working on https://github.com/scala/scala-lang/pull/1387, I have created a short blog post about a more time-sensitive matter - the release of Scala 3.1.3.

### CODE REVIEW

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+### Possibility to generate arbitrary class implementations in macros
+
+For a long time, generating arbitrary classes was not possible using the quotes api. With 3.1.3, the situation has changed. We added two missing blocks: experimental methods `ClassDef.apply` and `Symbol.newClass`. Now using them, you can write a snippet similar to the following:
+
+```scala
+import scala.quoted.*
+
+class Base
+
+transparent inline def classNamed(inline name: String) = new Base
```
Comment (prolativ) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:75]: `classNamedExpr` is unused here. I assume you meant something like
```scala
import scala.quoted.*

class Base

transparent inline def classNamed(inline name: String): Base = ${ classNamedExpr('name) }
def classNamedExpr(nameExpr: Expr[String])(using Quotes): Expr[Base] =
  import quotes.reflect.*

  val name = nameExpr.valueOrAbort
  val parents = List(TypeTree.of[Base])
  def decls(cls: Symbol) = List.empty  // put something interesting here
  val body = List.empty                // ...and here

  val symbol = Symbol.newClass(Symbol.spliceOwner, name, parents.map(_.tpe), decls, selfType = None)
  val classDef = ClassDef(symbol, parents, body)
  val ctor = Select(New(TypeIdent(symbol)), symbol.primaryConstructor)
  val newClass = Typed(Apply(ctor, Nil), TypeTree.of[Base])

  Block(List(classDef), newClass).asExprOf[Base]
```

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  val ctor = Select(New(TypeIdent(symbol)), symbol.primaryConstructor)
+  val newClass = Typed(Apply(ctor, Nil), TypeTree.of[Base])
+
+  Block(List(classDef), newClass).asExprOf[Base]
+```
+
+Then you can invoke the macro with
+
+```scala
+val impl: Base = scope.classNamed("Foo")
```
Comment (prolativ) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:95]: There's no `scope` in this example. Did you mean it to be the package where `classNamed` is defined? I would assume the entire example should work if e.g. someone pastes these two snippets to REPL one after another

Comment (Kordyjan) [reply blog/_posts/2022-06-15-scala-3.1.3-released.md:95]: Yup! Another leftover from me chasing this bug.

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.3 released!
+---
+
+We are happy to announce the release of Scala 3.1.3! You can read more about all the improvements and fixes in [the full changelog](https://github.com/lampepfl/dotty/releases/tag/3.1.3). We have also prepared a short highlights of the most exciting additions in the new version.
```
Comment (Philippus) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:8]:
```suggestion
We are happy to announce the release of Scala 3.1.3! You can read more about all the improvements and fixes in [the full changelog](https://github.com/lampepfl/dotty/releases/tag/3.1.3). We have also prepared a short list of highlights of the most exciting additions in the new version.
```

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  first[String]
+
+inline def second[A]: Int =
+  compiletime.summonInline[Foo[A]].foo
+
+inline def first[A]: Int =
+  second[A] + 42
+```
+
+will now report compilation error looking like this
```
Comment (Philippus) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:58]:
```suggestion
will now report a compilation error looking like this
```

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+![the new error]({{ site.baseurl }}/resources/img/inline-error-after.png)
+
+instead of previous
```
Comment (Philippus) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:62]:
```suggestion
instead of previously
```

**Review by Philippus — CHANGES_REQUESTED** [review]

File: blog/_posts/2022-06-15-scala-3.1.3-released.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.3 released!
+---
+
+We are happy to announce the release of Scala 3.1.3! You can read more about all the improvements and fixes in [the full changelog](https://github.com/lampepfl/dotty/releases/tag/3.1.3). We have also prepared a short highlights of the most exciting additions in the new version.
+
+## Highlights of the release
+
```
Comment (bishabosha) [inline blog/_posts/2022-06-15-scala-3.1.3-released.md:11]: https://github.com/lampepfl/dotty/pull/15275 - porting of scalac from bash to scala - will enable `cs setup` in future to match bash scalac command.

https://github.com/lampepfl/dotty/pull/15355 - windows batch files now match bash script launchers

Comment (Kordyjan) [reply blog/_posts/2022-06-15-scala-3.1.3-released.md:11]: I'm not sure if this is an essential change for most of our users. A lot of users are not aware of existence of launch scripts. Those who use them will also not see immediate improvements, as (if I remember correctly) this is just a new implementation with the same functionality.

The coursier part may be interesting, but it is still not implemented, so I wouldn't put that in the release notes.

**Review by prolativ — APPROVED** [review]

**Pull request merged**